1.0.1/jessie/product/README.md

```diff
@@ -17,3 +17,27 @@ This image contains:
 ## Supported tags
 
 - [`1.0.1`, `latest` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.0.1/jessie/product/Dockerfile)
+
+## Example Usage
+
+1. Create a Dockerfile for your application, the following example assumes you have already compiled your application (which is the expected use case for this image)
+
+  ```
+  FROM microsoft/aspnetcore
+  WORKDIR /app
+  COPY . /app
```

Comment:
hmmm. That's a fair call. I have always preferred being explicit about it, `COPY . .` seemed like crazy magic. But I totally buy your argument and think I'll change it.